public/main.js

```diff
@@ -199,6 +231,19 @@ let likeRecipe = (id) => {
     .then(res => console.log(res))
 }
 
+let showRecipesInNav = document.querySelector('.view-saved');
+showRecipesInNav.addEventListener('click', () => showLikedRecipes());
```

Comment:
We may want to move lines 234 and 235 inside the setUpEventListeners function just to help maintain consistency with the event listeners.